maven-sdk4/src/main/docker/Dockerfile

```diff
@@ -7,9 +7,10 @@ COPY Dockerfile extensions/*.jar $TOMCAT_DIR/webapps/alfresco/WEB-INF/lib/
 
 # Copy Dockerfile to avoid an error if no AMPs exist
 COPY Dockerfile extensions/*.amp $TOMCAT_DIR/amps/
+USER root
```

Comment:
Could you provide a link to the related GH issue to give some context why this is necessary?